62.CAD/fragment_shader.hlsl

```diff
@@ -214,23 +214,27 @@ struct ClippedSignedDistance
         float_t roundedDistance = closestDistanceSquared - thickness;
 
         // TODO[Przemek]: if style `isRoadStyle` is true use rectCapped, else use normal roundedDistance and remove this #ifdef
-#define ROUNDED
-#ifdef ROUNDED
-        return roundedDistance;
-#else
-        const float_t aaWidth = globals.antiAliasingFactor;
-        float_t rectCappedDistance = roundedDistance;
-
-        if (clipped)
+        if(!isRoadStyle)
         {
-            float_t2 q = mul(curve.getLocalCoordinateSpace(closestT), pos - curve.evaluate(closestT));
-            rectCappedDistance = capSquare(q, thickness, aaWidth);
+            return roundedDistance;
         }
         else
-            rectCappedDistance = rectCappedDistance;
+        {
+            const float_t aaWidth = globals.antiAliasingFactor;
+            float_t rectCappedDistance = roundedDistance;
 
-        return rectCappedDistance;
-#endif
+            if (clipped)
+            {
+                float_t2 q = mul(curve.getLocalCoordinateSpace(closestT), pos - curve.evaluate(closestT));
+                rectCappedDistance = capSquare(q, thickness, aaWidth);
+            }
+            else
+            {
+                rectCappedDistance = rectCappedDistance;
+            }
```

Comment:
Let's remove this redundant piece of code, IDK why it was originally there maybe I was doing some testing